README.md

```diff
@@ -1,13 +1,74 @@
-# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
 
-Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+<br />
+<p align="center">
+  <a href="https://github.com/pscott/TaskMaster">
+    <img src="assets/logo.png" alt="Logo" width="90" height="90">
+  </a>
 
----
+  <p align="center">
+    <img src="assets/title.png" alt="TaskMaster" width="280" height="70">
+  </p>
+
+  <p align="center">
+    A process control system for UNIX.
+    <br />
+    <a href="https://github.com/pscott/TaskMaster">View Demo</a>
+    ·
+    <a href="https://github.com/pscott/TaskMaster/issues">Report Bug</a>
+    ·
+    <a href="mailto:abarthel@student.42.fr">Request Feature</a>
+  </p>
+</p>
+
+[![Contributors][contributors-shield]][contributors-url]
+
+
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
```

Comment:
Those sub-titles are missing. Maybe we'd be better off creating them now so that we don't forget 'em. Btw, is there no way of automatically generating this Table Of Contents? :)

Reply:
Multiple sections will be coming depending on the need. Sure we could create few additional sections now - as you wish.

Reply:
What I wanted to say is that if we have sections in the TOC then they should appear in the readme. Creating sections in the TOC without the corresponding text section OR creating text sections without the corresponding TOC sections will probably lead to dead links / broken stuff in the future :p